libs/ha/ha_common_libs_k8s.py

```diff
@@ -1784,3 +1784,68 @@ def failover_pod(self, pod_obj, pod_yaml, failover_node):
                 return resp
 
         return True, f"Successfully failed over pods {list(pod_yaml.keys())}"
+
+    def iam_bucket_cruds(self, event, s3_obj, user_crud=False, num_users=None, bkt_crud=None,
```

Comment:
from libs.csm.csm_interface import csm_api_factory

self.csm_obj = csm_api_factory("rest")

self.csm_obj.verify_create_iam_user_rgw(user_type="valid") - Create and verify IAM user
return (
True/ False based on uer is create or not,  
response json which will contain access key and secret key
)

U can iterate above function based on number of users required. For now we are diverting S3 account creation to IAM account so that all the S3 tests works.

User management lies with CSM now, it will be good if we use them directly from CSM..

Reply:
@kachhwahadivya  Thanks, current function is using above mentioned function only.
@ankita-93 csm has single user creation, which we are already using. Also, we need to start these operations in background, son can't iterate over existing function.